USAGE.md

```diff
@@ -0,0 +1,561 @@
+# MITRE ATT&CK® Navigator Version 4.4
```

Comment:
We should probably remove the version numbers here. Previously we included them because they could inserted by Angular, but since they can't in the Markdown they're a likely thing to forget to update when releasing new versions of the application.